sync-diff-inspector/sync-diff-inspector-overview.md

```diff
@@ -222,6 +222,44 @@ Run the following command:
 
 This command outputs a check report to the log and describes the check result of each table. sync-diff-inspector generates the SQL statements to fix inconsistent data and stores these statements in a `fix.sql` file.
 
+#### Log
+
+The running sync-diff-inspector periodically prints the progress in log in the following format:
+
+```log
+[2020/11/12 17:47:00.170 +08:00] [INFO] [checkpoint.go:276] ["summary info"] [instance_id=target] [schema=test] [table=test_table] ["chunk num"=1000] ["success num"=80] ["failed num"=1] ["ignore num"=0]
+```
+
+- `chunk num`: The total number of chunks to be checked.
+- `success num`: The number of chunks that have been checked as consistent.
+- `failed num`: The number of chunks that fail the check. Check failure might be caused by an error or data inconsistency.
+- `ignore num`: The number of chunks that are ignored for the check. If the value of `sample-percent` is smaller than `100`, sync-diff-inspector checks data by sampling, where some chunks are ignored.
+
+#### Result
+
+After the check is finished, sync-diff-inspector outputs a report.
+
++ If data is consistent, a log example is as follows:
+
+    ```log
+    [2020/11/12 17:47:00.174 +08:00] [INFO] [report.go:80] ["check result summary"] ["check passed num"=1] ["check failed num"=0]
+    [2020/11/12 17:47:00.174 +08:00] [INFO] [report.go:87] ["table check result"] [schema=test] [table=test_table] ["struct equal"=true] ["data equal"=true]
+    [2020/11/12 17:47:00.174 +08:00] [INFO] [main.go:75] ["check data finished"] [cost=353.462744ms]
+    [2020/11/12 17:47:00.174 +08:00] [INFO] [main.go:69] ["check pass!!!"]
+    ```
+
++ If data is inconsistent or an error occurs, a log example is as follows:
```

Comment:
how about update `an error` to `some errors`?

Reply:
Updated in https://github.com/pingcap/docs/pull/4247/commits/75c81b628a73ae6910114d41bbe9966cdc9cfb45.